node/manifest.ts

```diff
@@ -90,6 +91,16 @@ const addExcludedPatterns = (
   }
 }
 
+/**
+ * Turns 'get' into ['GET']
+ */
+const normalizeMethods = (method: undefined | string | string[]) => {
+  if (!method) return
+
+  const methods = Array.isArray(method) ? method : [method]
+  return methods.map((method) => method.toUpperCase())
```

Comment:
We take this from user code that might not even be typed, so if someone uses a non-string value in the `method` property, we'll throw a bit of a cryptic error when trying to call `.toUpperCase()`?

Reply:
good catch! added a type check in d9c00080dfd053d388748948a5311ba14eceedbf.